designs/administrator/future-openshift/user-preferences/index.md

```diff
@@ -0,0 +1,62 @@
+---
+parent: Administrator
+version: 4.9
+---
+
+# Adding User Preferences to OpenShift Console
+
+We are finally adding user preferences to the console in 4.9. This enhancement will allow individual users to set preferences across the console, providing a customizable experience.
+
+In 4.9 user preferences will consist of two tabs:
+1. General preferences
+2. Language and region preferences
+
+All changes in user preferences will be autosaved.
+
+## Accessing user preferences
+
+![Masthead](img/2.png)
+- To access user preferences, navigate to the user dropdown in the masthead and click on **User preferences**.
+
+## General preferences
+
+![General-default](img/3.png)
+- General preferences will enable users to customize defaults for the **Perspective**, **Project**, **Topology**, and their preferred way to **Create/Edit resources**.
```

Comment:
I think it could be inferred but somehow the absence of the word "Default" in front of each of these headings (Default perspective, Default project, etc) leaves me momentarily wondering what they represent? Especially when the value is "last viewed" since I am not sure yet exactly what that is referring to. The helper text under the field makes it clear but I guess was sort of wondering if it could be surfaced sooner. Maybe even calling this setting page "Defaults" or something?

Also I sort of feel like "Create/Edit resource" needs one more word like "method" to indicate what about creating a resource you are setting, especially when the options are slightly obscured by the default value of 'Last viewed.' I could almost think that the dropdown would contain a bunch of resources.

Reply:
Would it be worth adding that to the description at the top of the page like set your individual default preferences? 

Good point about the "method" - I will add that and update the wireframes.

Reply:
Yeah I think that could be a fine solution and include "set your individual defaults and preferences" or something, or could just stay as-is, not a huge issue.